source/includes/_payment.md

```diff
@@ -3643,41 +2429,17 @@ paymentApi.create_payment_tags(payment_id,
 
 > Example Response:
 
-```shell
-# Subset of headers returned when specifying -v curl option
-< HTTP/1.1 201 Created
-< Location: http://127.0.0.1:8080/1.0/kb/payments/8fe697d4-2c25-482c-aa45-f6cd5a48186d/
-< Content-Type: application/json
-< Content-Length: 0
-```
-
-```java
-//First element of the list
-class Tag {
-    org.killbill.billing.client.model.gen.Tag@bd138472
-    tagId: 1bb4b638-3886-4f73-90a5-89eb6d1bcf7f
-    objectType: PAYMENT
-    objectId: 917992d3-5f1f-4828-9fff-799cc4211aa9
-    tagDefinitionId: 353752dd-9041-4450-b782-a8bb03a923c8
-    tagDefinitionName: foo
-    auditLogs: []
+```json
+{
+  "tagId":"a46cfeb6-e175-42db-be62-7f117326ab4e",
+  "objectType":"PAYMENT",
+  "objectId":"28af3cb9-275b-4ac4-a55d-a0536e479069",
+  "tagDefinitionId":"353752dd-9041-4450-b782-a8bb03a923c8",
+  "tagDefinitionName":"foo",
+  "auditLogs":[]
```

Comment:
Same comment as above, I think an empty body is sent, could you please confirm and update if required?